heron/tools/ui/resources/templates/shell.snip.html

```diff
@@ -1,3 +1,22 @@
+<!--
```

Comment:
This is used as a Jinja2 template, so it would be preferable to make it a [Jinja2 comment](https://jinja.palletsprojects.com/en/2.10.x/templates/#comments)

Reply:
I assume this is true for all of the files in that folder?

Reply:
Yep. Switching those would be nice too - the notice wasn't converted in the FastAPI+Jinja2 upgrade

Reply:
Thanks for catching @Code0x58